packages/svelte/src/server/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import type { RenderOutput } from '#server';
+import type { ComponentProps, Component, SvelteComponent, ComponentType } from 'svelte';
+
+/**
+ * Only available on the server and when compiling with the `server` option.
+ * Takes a component and returns an object with `body` and `head` properties on it, which you can use to populate the HTML when server-rendering your app.
+ */
+export function render<
```

Comment:
had to move this into a `d.ts` file - JSDoc was too limiting to properly type this without type errors